### PR TITLE
Add help text to not self explaining fields

### PR DESCRIPTION
**Pull request by sliverc**

### CODE REVIEW

File: caluma/workflow/models.py [hunk 1]
```diff
@@ -39,7 +43,12 @@ class Workflow(SlugModel):
     meta = JSONField(default=dict)
     is_published = models.BooleanField(default=False)
     is_archived = models.BooleanField(default=False)
-    start = models.ForeignKey(Task, on_delete=models.CASCADE, related_name="+")
+    start = models.ForeignKey(
+        Task,
+        on_delete=models.CASCADE,
+        related_name="+",
+        help_text="Task workflow will be started with.",
```
Comment (czosel) [inline caluma/workflow/models.py]:
```suggestion
        help_text="First task of the workflow.",
```

**Comment by sliverc** [comment]
@czosel 
Adjusted so group jexl doc actually appears now in all mutations

**Comment by sliverc** [comment]
@czosel CI fixed ready for review.

File: caluma/workflow/models.py (hunk 1 above)
Comment (czosel) [reply caluma/workflow/models.py]: How about the text change?

Comment (sliverc) [reply caluma/workflow/models.py]: ouch with my force pushed I have lost this change... :smile: Let me readd it again.

Comment (sliverc) [reply caluma/workflow/models.py]: done

**Pull request merged**